### PR TITLE
feat: add a petstore by name

### PR DESCRIPTION
**Pull request by BelfordZ**

### CODE REVIEW

**Comment by shanejonas** [comment]
need to add this to `index.js`

File: index.js [hunk 1]
```diff
@@ -3,5 +3,6 @@ module.exports = {
   links: require('./service-descriptions/link-example-openrpc.json'),
   petstore: require('./service-descriptions/petstore-openrpc.json'),
   petstoreExpanded: require('./service-descriptions/petstore-expanded-openrpc.json'),
-  simpleMath: require('./service-descriptions/simple-math-openrpc.json')
+  simpleMath: require('./service-descriptions/simple-math-openrpc.json'),
+  petstore: require('./service-descriptions/params-by-name-petstore-openrpc.json')
```
Comment (shanejonas) [inline index.js]: cant have 2 keys named `petstore`

Comment (BelfordZ) [inline index.js]:
```suggestion
  petstoreByName: require('./service-descriptions/params-by-name-petstore-openrpc.json')
```

File: service-descriptions/params-by-name-petstore-openrpc.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          "name": "petId",
+          "required": true,
+          "description": "The id of the pet to retrieve",
+          "schema": {
+            "type": "string"
+          }
+        }
+      ],
+      "result": {
+         "name": "pets",
```
Comment (shanejonas) [inline service-descriptions/params-by-name-petstore-openrpc.json]:
```suggestion
        "name": "pets",
```

**Pull request merged**